dashboard/test/controllers/transfers_controller_test.rb

```diff
@@ -3,284 +3,228 @@
+    post :create, params: @params
+    assert_response 204
+    refute Follower.exists?(student_user: @word_student, section: @word_section)
   end
 
   test "transferring to the same section does nothing" do
-    post :create, params: {
-      new_section_code: @word_section.code,
-      student_ids: @word_user_1.id.to_s
-    }
-    assert Follower.exists?(student_user: @word_user_1, section: @word_section)
-  end
-
-  test "transferring a student to a section owned by a teacher the student "\
-       "already follows causes the student to transfer sections under that teacher" do
-    post :create, params: {
-      new_section_code: @picture_section.code,
-      student_ids: @word_user_1.id.to_s
-    }
-    assert_not Follower.exists?(student_user: @picture_user_1, section: @picture_section)
-  end
-
-  test "transferring to a new teacher causes a student to join the section" do
-    new_teacher = create(:teacher)
-    new_word_section = create(:section, user: new_teacher, login_type: 'word')
-
-    post :create, params: {
-      new_section_code: new_word_section.code,
-      student_ids: @word_user_1.id.to_s,
-      current_section_code: @word_section.code,
-      stay_enrolled_in_current_section: false
-    }
-    assert Follower.exists?(student_user: @word_user_1, section: new_word_section)
-  end
-
-  test "transferring to a new teacher does not modify existing sections for a student" do
-    new_teacher = create(:teacher)
-    new_word_section = create(:section, user: new_teacher, login_type: 'word')
-
-    post :create, params: {
-      new_section_code: new_word_section.code,
-      student_ids: @word_user_1.id.to_s
-    }
-
-    assert Follower.exists?(student_user: @word_user_1, section: @word_section)
-  end
+    @params[:new_section_code] = @params[:current_section_code]
+    post :create, params: @params
+    assert_response 400
+    assert Follower.exists?(student_user: @word_student, section: @word_section)
+  end
+
+  # TODO(asher): Determine what the desired behavior is, uncommenting this test
+  # or fixing the controller code appropriately.
+  # test "transferring a student to a section owned by a teacher the student "\
+  #      "already follows causes the student to transfer sections under that teacher" do
+  #   existing_section = create :section, user: @other_teacher, login_type: 'word'
+  #   Follower.create!(
+  #     user: @other_teacher,
+  #     student_user: @word_student,
+  #     section: existing_section
+  #   )
+  #   @params[:new_section_code] = @other_teacher_section.code
+  #   @params[:stay_enrolled_in_current_section] = false
+  #   post :create, params: @params
+  #
+  #   assert_response 204
+  #   refute Follower.exists?(student_user: @word_student, section: existing_section)
+  #   assert Follower.exists?(student_user: @word_student, section: @other_teacher_section)
+  # end
 
   test "transferring a student with a messed up email succeeds" do
```

Comment:
Though the post failed (not having a `current_section_code`, the `assert` was satisfied because `@picture_user_1` was a member of `@word_section` via `setup`.